api/src/test/java/org/pmiops/workbench/api/UserControllerTest.java

```diff
@@ -64,6 +67,8 @@
   private static long incrementedUserId = 1;
   private static final Cloudbilling testCloudbilling = TestMockFactory.createMockedCloudbilling();
 
+  @Mock private static FreeTierBillingService testFreeTierBillingService;
```

Comment:
nit: existing pattern in this file would be to add this in `@MockBean`, then `@autowire` it in below. Ideally this is named mockFreeTierBillingService.

ALl good candidates for a style document. Happy to write this one

Reply:
Thanks - I would appreciate that.  I get lost in the many ways we can mock things.